### PR TITLE
feat: Add new payload examples for various events

### PR DESCRIPTION
**Pull request by wolfy1339**
Adds some new payload examples

### CODE REVIEW

File: payload-examples/api.github.com/repository.edited.2.payload.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
+{
```
Comment (wolfy1339) [inline payload-examples/api.github.com/repository.edited.2.payload.json:1]: I wasn't sure what to call this one. Should I name it `repository.edited.with-default_branch-edit.payload.json`

File: payload-examples/api.github.com/pull_request.assigned.payload.json [hunk 2]
```diff
@@ -366,6 +366,7 @@
       }
     },
     "author_association": "OWNER",
+    "active_lock_reason": null,
```
Comment (wolfy1339) [inline payload-examples/api.github.com/pull_request.assigned.payload.json:369]: This is either `resolved`, `off-topic`, `too heated`, `spam` when it's locked and `null` when it's unlocked

**Comment by wolfy1339** [comment]
I think I'll leave it as-is for now. I don't want it to get too big

File: payload-examples/api.github.com/repository.edited.2.payload.json (hunk 1 above)
Comment (gr2m) [reply payload-examples/api.github.com/repository.edited.2.payload.json:1]: yeah I think the more descriptive the better

File: payload-examples/api.github.com/pull_request.assigned.payload.json (hunk 2 above)
Comment (gr2m) [reply payload-examples/api.github.com/pull_request.assigned.payload.json:369]: let's update that in the schema at #186 before we merge the PR

Comment (wolfy1339) [reply payload-examples/api.github.com/pull_request.assigned.payload.json:369]: I've done that at https://github.com/heuels/webhooks/commit/8124337883c5937c688c3d57a24f6db25651b8c6

**Comment by wolfy1339** [comment]
@gr2m I've addressed your feedback. Anything else?

**Review by gr2m — APPROVED** [review]
Thanks!

**Pull request merged**